lib/Models/ImageryLayerCatalogItem.js

```diff
@@ -44,7 +44,7 @@ var ImageryLayerCatalogItem = function(terria) {
     /**
      * Gets or sets the opacity (alpha) of the data item, where 0.0 is fully transparent and 1.0 is
      * fully opaque.  This property is observable.
-     * @type {Number}
+     * @type {Number|String}
```

Comment:
And https://github.com/terriajs/terriajs/blob/0615a601b87538362a2617d1bf87797518111449/wwwroot/test/init/composite.json#L29-L29

And https://github.com/terriajs/terriajs/blob/0615a601b87538362a2617d1bf87797518111449/wwwroot/test/init/test-tablestyle.json#L18-L18